src/components/general/SideSheet/Standard/Standard.style.ts

```diff
@@ -0,0 +1,168 @@
+import { makeStyles, createStyles } from '@equinor/fusion-react-styles';
+
+export const useStyles = makeStyles(
+    createStyles({
+        container: {
+            position: 'relative',
+            zIndex: 1,
+            boxSizing: 'border-box',
+            height: '100%',
+            borderLeft: '2px solid var(--color-black-alt4)',
+            transition: 'width 0.1s',
+            overflow: 'visible',
+            color: 'var(--color-primary-accent)',
+            width: 'var(--sidesheet-size)',
+            flexShrink: 0,
+            background: 'var(--color-white)',
+            display: 'flex',
+            flexDirection: 'column',
+
+            '@media (max-width: 767px)': {
+                width: '100%',
+                height: '100%',
+                transition: 'transform 0.1s ease-out',
+
+                '&:not(& $isCollapsed)': {
+                    width: '100% !important',
+                    left: 0,
+                    top: 0,
+                    bottom: 0,
+                    right: 0,
+                    position: 'fixed',
+                    pointerEvents: 'all',
+                },
+
+                '& $resizeHandle': {
+                    display: 'none',
+                },
+            },
+        },
+        screenPlacementLeft: {
+            borderLeft: 'none',
+            borderRight: '2px solid var(--color-black-alt4)',
+
+            '& $header': {
+                flexDirection: 'row-reverse',
+            },
+
+            '& $resizeHandle': {
+                right: 'calc(var(--grid-unit) * -2px)',
+                left: 'auto',
+            },
+        },
+        xlarge: {
+            '--sidesheet-size': 'calc(var(--grid-unit) * 80px)',
+        },
+
+        large: {
+            '--sidesheet-size': 'calc(var(--grid-unit) * 60px)',
+        },
+
+        medium: {
+            '--sidesheet-size': 'calc(var(--grid-unit) * 40px)',
+        },
+
+        small: {
+            '--sidesheet-size': 'calc(var(--grid-unit) * 30px)',
+        },
+
+        content: {
+            display: 'flex',
+            flexDirection: 'column',
+            overflow: 'auto',
+            flexGrow: 1,
+        },
+        header: {
+            display: 'flex',
+            alignItems: 'center',
+            overflow: 'hidden',
+            flexShrink: 0,
+        },
+        collapseButtonContainer: {
+            width: 'calc(var(--grid-unit) * 6px)',
+            height: 'calc(var(--grid-unit) * 6px)',
+            borderBottom: '1px solid var(--color-black-alt4)',
```

Comment:
This line `borderBottom: '1px solid var(--color-black-alt4)'` should be removed. 

I have a PR on this that removes it from the less files.